src/components/list-section/index.tsx

```diff
@@ -24,6 +22,12 @@ const services = [
     and implement streamlined workflows that increase productivity and reduce costs.
     Our approach ensures that your business runs smoothly and is positioned for sustainable growth.`,
   },
+  {
+    title: `Kanban Board`,
+    description: `Visualize your workflow and enhance team collaboration with our Kanban Board.
+    This flexible tool supports Scrum and Kanban methodologies, making it easy to adapt to your team's needs.
+    Organize tasks, track progress, and identify bottlenecks early to keep your projects on track and under control.`,
```

Comment:
Our HR tools streamline your people management processes, helping you build a strong, motivated team. The Employee Evaluation feature enables fair, structured assessments, fostering growth and accountability. With the HR Management Plan, you can align your people strategies with business goals, ensuring smooth operations and a supportive workplace culture. The Recruitment tool simplifies talent acquisition, allowing you to find and onboard the right people efficiently. Finally, our Retention Plan provides actionable insights and strategies to keep your top talent engaged and committed, reducing turnover and strengthening team stability.